camera_ids:
  - 218622273385
  - 218622276213
  - 218622270352
 
#not sure which id is which camera, need to verify after mounting
camera_218622273385: #cam T
  offset_x: 395.716
  offset_y: 131.1
  offset_z: 0.0
  rx: 180
  ry: 0.0
  rz: 0.0
 
camera_218622276213: # cam A
  offset_x: 0.0
  offset_y: 131.1
  offset_z: 94.9
  rx: -120
  ry: 0.0
  rz: 0.0
 
camera_218622270352: # cam B
  offset_x: 0.0
  offset_y: 131.1
  offset_z: 498.7
  rx: -60.0
  ry: 0.0
  rz: 0.0
 
p_dist: 73.884 # distance from servo to cam A and B axis. add to x offset if rz = 0, subtract from -x offset if rz = 180, or do trigonometry